src/ssl.c

```diff
@@ -14624,15 +14628,15 @@ int wolfSSL_CIPHER_get_auth_nid(const WOLFSSL_CIPHER* cipher)
         {"SRP",     NID_auth_srp},
         {"ECDSA",   NID_auth_ecdsa},
         {"None",    NID_auth_null},
-        {NULL,      NID_undef}
+        {NULL,      wc_NID_undef}
```

Comment:
Change all NIDs over.
NID_auth_ecdsa is definitely in OpenSSL.

Reply:
OK yeah good idea, close the deal.  the ones done in the first one were just what was necessary to get `OPENSSL_EXTRA` clean.